tools/autodeployment/src/sibling-checker/index.js

```diff
@@ -0,0 +1,80 @@
+/* eslint-disable no-use-before-define */
+const fetch = require('node-fetch');
+const { setIntervalAsync, clearIntervalAsync } = require('set-interval-async/dynamic');
+
+const server =
+  process.env.DEPLOY_TYPE === 'production'
+    ? 'https://telescope.cdot.systems/health.'
+    : 'https://dev.telescope.cdot.systems/health.';
```

Comment:
What I meant with the sibling checker approach is that, in this section where you decide what URL to use, you could use `dev.telescope` for the production machine and vice versa.
This way, the checks would be run against *the other machine*, and we'd be able to get an answer even if the targeted machine is down (if the machine checking is also the machine being checked, you won't get any messages when the machine is down).

Reply:
You are right, this is the idea. I need to invert the URLs!